artifactor/plugins/reporter.py

```diff
@@ -262,7 +268,7 @@ def build_dict(self, path, container, contents):
             if head not in container['_sub']:
                 container['_sub'][head] = deepcopy(_tests_tpl)
             # Call again to recurse down the tree.
-            self.build_dict('/'.join(end), container['_sub'][head], contents)
+            self.build_dict(end, container['_sub'][head], contents)
```

Comment:
The code I wrote does not care about the leading slashes, it strips them away as the first step.